content/en/apps/guides/messaging/rapidpro.md

```diff
@@ -0,0 +1,114 @@
+---
+title: RapidPro SMS Gateway
+linkTitle: "RapidPro"
+weight:
+description: >
+    Integration for sending and receiving SMS
+relatedContent: >
+    apps/reference/app-settings/sms
+    apps/guides/forms/app-form-sms
+    apps/guides/messaging/sms-states
+    apps/guides/messaging/message-loops
+    apps/guides/messaging/shortcodes
+---
+
+
+As of v3.11.0, SMS messages can be sent and received using [RapidPro](https://community.rapidpro.io/).
+
+## RapidPro configuration
+
+### Store globals
+
+Generate a long unique key to use as the `cht_api_key`.
+
+Log in to your RapidPro dashboard, go to the globals page (`/globals/`) and create two globals with the following data:
+
+- name: `cht_url`, value: `https://<your-cht-instance-host>/api/v1/sms/radpidpro/incoming-messages`. The instance host should not include basic authentication.
```

Comment:
I assume we don't want to include basic auth because then the admin password will be stored in plaintext in rapidpro somewhere. If so, I think we can use stronger wording here.

```suggestion
- name: `cht_url`, value: `https://<your-cht-instance-host>/api/v1/sms/radpidpro/incoming-messages`. For security the instance host **must** not include basic authentication.
```